lib/mix/lib/mix.ex

```diff
@@ -161,7 +161,7 @@ defmodule Mix do
 
     * `MIX_ARCHIVES` - specifies the directory into which the archives should be installed
     * `MIX_DEBUG` - outputs debug information about each task before running it
-    * `MIX_ENV` - specifies which environment should be used. see Environments
+    * `MIX_ENV` - specifies which environment should be used. See [Environments](#environments)
```

Comment:
Are you sure this link works? When I click on "Environments" in the [current docs for `Mix`](http://elixir-lang.org/docs/stable/mix/Mix.html) I get this link: http://elixir-lang.org/docs/stable/mix/Mix.html#module-environments, so `#module-environments` instead of `#environments`.